roles/storwize/tasks/main.yml

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
We should re-work this logic. Reading seems like we want to generate ssh keys and add the pub key to v7k and in addition creating the pool. What user are these keys for?
We probably should do following:
- have private and pub key part of envs, see swifttool example: https://github.com/blueboxgroup/ursula/blob/master/envs/example/ci-ceph-swift-rhel/group_vars/all.yml#L110
- This will enable us to get this handle as pre-req for case where we will support BOY V7K.